### PR TITLE
Updates to VCFFilter documentation and filter field

### PR DESCRIPTION
**Pull request by VJalili**
This PR makes the following updates to the VCFFilter:
1. Update the documentation of VCFFilter for more intuitive description of how to specify filters;
2. Ignore `"` (double-quotation) and `'` (single-quotation) for the filter field (if given, the wrapper removes the quotations);
3. Require at least one filter.

### CODE REVIEW

**Comment by bgruening** [comment]
Thanks @VJalili!

File: tool_collections/vcflib/vcffilter/vcffilter.xml [hunk 1]
```diff
@@ -30,7 +31,27 @@ input1.vcf.gz
...
+                </valid>
+                <mapping initial="none">
+                    <add source="&apos;" target=""/>
+                    <add source="&quot;" target=""/>
+                </mapping>
+            </sanitizer>
+        </param>
+
+        <!-- Additional filters are allowed. -->
+        <repeat name="filter_repeat" title="more filters">
```
Comment (bgruening) [inline tool_collections/vcflib/vcffilter/vcffilter.xml]: @VJalili you can add `min="1"` to a repeat, this means a user is forced to put in at least one iteration.

Comment (VJalili) [reply tool_collections/vcflib/vcffilter/vcffilter.xml]: Yes, that sounds much better. Thanks!

**Comment by jennaj** [comment]
This should cover all the basic things we wanted to address. 

 - Help that matches updated input form !! (much clearer now)
 - Stripping quotes (users enter these by mistake due to older tutorial/form structure)
 - Requiring at least one filter term before job submission (users sometimes miss entering anything then not knowing what went wrong)

Thank you!

For reference, this is where we are tracking the fix all the way to usegalaxy.org: galaxyproject/usegalaxy-playbook#182

**Comment by bgruening** [comment]
@VJalili please also increase the tool version.

**Comment by VJalili** [comment]
I guess you're referring to the [wrapper version](https://github.com/galaxyproject/tools-iuc/blob/91a4a5aacf1e8bac5ddccb8504e46dae13b594aa/tool_collections/vcflib/macros.xml#L13). It says `rc`, not sure how we version these, but does `1.0.1` sound reasonable?

**Comment by bgruening** [comment]
Thanks @VJalili and sorry for being so slow these days.

**Comment by VJalili** [comment]
@bgruening Thank you for checking this.

**Pull request merged**